resources/language/resource.language.pl_pl/strings.po

```diff
@@ -180,3 +180,7 @@ msgstr "Klucz API YouTube"
 msgctxt "#30208"
 msgid "Enable live streams"
 msgstr "Włącz transmisje na żywo"
+
+msgctxt "#30209"
+msgid "Filter out shorts"
+msgstr "Odfiltruj #szorty"
```

Comment:
@hevet Can you provide a better translation?

Reply:
@mintsoft Odfiltruj krótkie.